certified-connectors/PowerBI Robots/apiDefinition.swagger.json

```diff
@@ -0,0 +1,326 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "DevScope PowerBI Robots",
```

Comment:
@AntonioArantesDevscope Indentation - The properties are not aligned properly and there is longer tab index between the properties. Please refer the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
identation resolved at commit "Json Identation"